Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -40,6 +40,7 @@ public SiteExtensionInfo(SiteExtensionInfo info)
             LocalIsLatestVersion = info.LocalIsLatestVersion;
             LocalPath = info.LocalPath;
             InstalledDateTime = info.InstalledDateTime;
+            InstallationArgs = info.InstallationArgs;
```

Comment:
No, I think this will end up being a no op if the version is the same. Seems like an edge case.